clog/loggers.py

```diff
@@ -269,7 +269,7 @@ def close(self):
             self.stream_files[name].close()
 
     def _create_file(self, stream):
-        return open(os.path.join(config.log_dir, stream + '.log'), 'ab', 0)
+        return open(os.path.join(config.log_dir.value, stream + '.log'), 'ab', 0)
```

Comment:
Since `config.log_dir` is set via `clog_namespace.get_string`, when would this be something other than a ValueProxy? Seems like even with a default it always returns the same type.